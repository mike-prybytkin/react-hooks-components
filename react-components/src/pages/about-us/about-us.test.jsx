import React from 'react';
import { render, screen } from '@testing-library/react';
import AboutUs from './about-us';
import mockText from '../../mocks/text';

const setUp = () => render(<AboutUs heading={mockText.headingAboutUs} />);

describe('About Us component', () => {
  it('should correctly render heading', () => {
    setUp();
    const aboutUsHeading = screen.getByText(new RegExp(mockText.headingAboutUs, 'i'));
    expect(aboutUsHeading).toBeInTheDocument();
  });

  it('should create snapshot', () => {
    const { asFragment } = setUp();
    expect(asFragment()).toMatchSnapshot();
  });
});
