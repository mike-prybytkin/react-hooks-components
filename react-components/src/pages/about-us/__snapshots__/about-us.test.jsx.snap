// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Us component should create snapshot 1`] = `
<DocumentFragment>
  <h1>
    About Us!
  </h1>
  <p
    class="sorry-message"
  />
</DocumentFragment>
`;
